M2/Macaulay2/packages/SpectralSequences.m2

```diff
@@ -249,21 +148,20 @@ min FilteredComplex := K -> min spots K
 support FilteredComplex := List => (
      K -> sort select (spots K, i -> K#i != 0))
 
-
 FilteredComplex _ InfiniteNumber :=
-FilteredComplex _ ZZ := ChainComplex => (K,p) -> (
+FilteredComplex _ ZZ := Complex => (K,p) -> (
   if K#?p then K#p 
   else if p < min K then K#(min K) 
   else if p > max K then K#(max K)
   )
 
 FilteredComplex ^ InfiniteNumber :=
-FilteredComplex ^ ZZ := ChainComplex => (K,p) -> K_(-p)
+FilteredComplex ^ ZZ := Complex => (K,p) -> K_(-p)
 
-chainComplex FilteredComplex := ChainComplex => K -> K_infinity
+chainComplex FilteredComplex := Complex => K -> K_infinity
```

Comment:
I believe the build error is coming from this line.  This should probably be `complex` instead of `chainComplex`.  (Since very recently, `chainComplex` isn't defined as a method function in Core any more.)